ui/apps/platform/src/Components/ReportJob/JobStatusPopoverContent.tsx

```diff
@@ -1,7 +1,55 @@
 import React from 'react';
 import { Flex, FlexItem } from '@patternfly/react-core';
+import { ReportJobStatus } from './types';
 
-function JobStatusPopoverContent() {
+export type JobStatusPopoverContentProps = {
+    statuses: ReportJobStatus[];
+};
+
+const contentMap: Record<ReportJobStatus, { label: string; description: string }> = {
+    WAITING: {
+        label: 'Waiting',
+        description: 'The report job is in the queue.',
+    },
+    PREPARING: {
+        label: 'Preparing',
+        description: 'The report job is being processed.',
+    },
+    DOWNLOAD_GENERATED: {
+        label: 'Report ready for download',
+        description: 'The report is ready and available for download.',
+    },
+    PARTIAL_SCAN_ERROR_DOWNLOAD: {
+        label: 'Partial report ready for download',
+        description: 'A report is partially complete and ready for download.',
+    },
+    EMAIL_DELIVERED: {
+        label: 'Report successfully sent',
+        description: 'The report was successfully emailed.',
+    },
+    PARTIAL_SCAN_ERROR_EMAIL: {
+        label: 'Partial report successfully sent',
+        description: 'A report is partially complete and was successfully emailed.',
+    },
+    ERROR: {
+        label: 'Report failed to generate',
+        description: 'There was an issue with the report job. Hover to view the error message.',
```

Comment:
Nit, not for now though: Hiding the error message behind a hover isn't an ideal UX IMO. If we have to spell it out in text that the user needs to hover to see the error, we should just display the error outright. Doubly-so that this hover message is in the popup, separate from the element that contains the error message that needs to be hovered.

Reply:
+1